.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+  push:
+    branches: master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js v14
```

Comment:
Currently the `.nvmrc` for the repo is on v12.8.2 so can we use node 12.x instead? https://github.com/square/maker/blob/master/.nvmrc

Actually, what would probably be best is if we could have the job run `nvm use` so we don't have to update it when we update `.nvmrc`.